statsmodels/distributions/copula/_special.py

```diff
@@ -103,8 +102,16 @@ def lin(n, z):
     if np.size(z) > 1:
         z = np.array(z)[..., None]
 
-    k = np.arange(n+1)
+    k = range(n + 1)
```

Comment:
this will be an iterator and not a list

Reply:
Yes, it is the same problem like in gumbel pdf (see comment) because the elemnts of np.arange are int32 and therefore the cache of sterling2, too, which quickly leads to overflow.